### PR TITLE
Chinese Update

### PR DESCRIPTION
**Pull request by Dissyu**
Chinese Update

### CODE REVIEW

File: addons/ai_hearing/stringtable.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (dedmen) [inline addons/ai_hearing/stringtable.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
```

File: addons/backpacks/stringtable.xml [hunk 2]
```diff
@@ -1,9 +1,11 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (dedmen) [inline addons/backpacks/stringtable.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
```

File: addons/core/stringtable.xml [hunk 3]
```diff
@@ -1,29 +1,37 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (dedmen) [inline addons/core/stringtable.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
```

File: addons/core/stringtable.xml [hunk 4]
```diff
@@ -347,10 +371,16 @@
...
                 <Original>Intercom Ducking</Original>
                 <Japanese>インターコムを絞る</Japanese>
+                <English>Intercom Ducking</English>
+                <Chinese>對講機自動迴避音量</Chinese>
+                <Chinesesimp>对讲机自动回避音量</Chinesesimp>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_intercomDucking_desc">
                 <Original>Intercom volume reduction on incoming vehicle radio transmission</Original>
                 <Japanese>車載無線機で受信中はインターコムの音量を絞ります</Japanese>
+                <English>Intercom volume reduction on incoming vehicle radio transmission</English>
```
Comment (dedmen) [inline addons/core/stringtable.xml]: English can be left away, game will default to Original. And that way we don't have duplicates for every string

File: addons/ai_hearing/stringtable.xml (hunk 1 above)
Comment (Dissyu) [reply addons/ai_hearing/stringtable.xml]: OK

File: addons/backpacks/stringtable.xml (hunk 2 above)
Comment (Dissyu) [reply addons/backpacks/stringtable.xml]: ok

File: addons/core/stringtable.xml (hunk 3 above)
Comment (Dissyu) [reply addons/core/stringtable.xml]: ok

**Comment by dedmen** [comment]
You didn't allow editing by maintainers, I can't fix the issues with this PR and can't merge it like this.

**Comment by dedmen** [comment]
No activity or attempt to fix these issues after over 2 years.

**Pull request closed without merging**